lib/chat/user_message_rows.dart

```diff
@@ -90,65 +77,100 @@ buildMessageRows(
-                children: <Widget>[
-                  FormatText(
-                    text: recipient.fullName,
-                    fontSize: 14.0,
-                  ),
-                  RichText(
-                    overflow: TextOverflow.ellipsis,
-                    text: TextSpan(
-                      style: GoogleFonts.muli(
-                        textStyle: TextStyle(
-                          color: Colors.grey,
-                          letterSpacing: .5, 
-                          fontSize: 12.0, 
-                        )
-                      ),
-                      children: [
-                        if(isSent != false) WidgetSpan(
-                          child: Container(
-                            margin: EdgeInsets.only(right: 6.0),
-                            child: Icon(
-                              Icons.send, 
-                              size: 12, 
-                              color: Colors.grey
-                            )
-                          ),
+              ),
+              Expanded(
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.start,
+                  children: <Widget>[
+                    Row(
+                      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                      children: <Widget>[
+                        FormatText(
+                          text: recipient.fullName,
+                          fontSize: 14.0,
                         ),
-                        TextSpan(
-                          text: lastMessageType == 0 ? 
-                            lastMessage.content : 
-                            imageMessageDesciption
+                        Container(
+                          padding: EdgeInsets.only(right: 10.0),
+                          child: FormatText(
+                            text: DateFormat('jm').format(
+                              DateTime.fromMillisecondsSinceEpoch(
+                                int.parse(lastMessage.timestamp)
+                              )
+                            ),
+                            fontSize: 14.0,
+                            fontWeight: FontWeight.w400,
+                            textColor: Colors.grey,
+                          )
                         )
-                      ]
+                      ],
                     ),
-                  )
-                ]
+                    RichText(
+                      overflow: TextOverflow.ellipsis,
+                      text: TextSpan(
+                        style: GoogleFonts.muli(
+                          textStyle: TextStyle(
+                            color: Colors.grey,
+                            letterSpacing: .5, 
+                            fontSize: 12.0, 
+                          )
+                        ),
+                        children: [
+                          if(isSent != false) WidgetSpan(
```

Comment:
I think since this is a multi-line widget below the if, it'd be a good idea to include brackets to wrap around the widgets controlled by the condition.

Reply:
I think the brackets around the widget actually returns them as a Set. I'll double check this again when its not 2:30 in the morn